test/unit/foreman/renderer/snapshots/ProvisioningTemplate/cloud-init/CloudInit_default.host4dhcp.snap.txt

```diff
@@ -142,7 +142,9 @@ runcmd:
     
     
       
-    
+      subscription-manager refresh
```

Comment:
The spaces indicate that a previous command leaves traces. While in this scenario it's not critical, it would be nice if you could trace it down and make sure there are no whitespaces emitted.